Source/Xamarin/Prism.DryIoc.Forms/DryIocContainerExtension.cs

```diff
@@ -49,16 +61,15 @@ public object Resolve(Type type, string name)
             return Instance.Resolve(type, serviceKey: name);
         }
 
-        public object ResolveViewModelForView(object view, Type viewModelType)
+        public virtual object ResolveViewModelForView(object view, Type viewModelType)
```

Comment:
why is this being made virtual?

Reply:
there are some edge cases where advanced users may need to inject some custom logic in here, and you shouldn't have to implement the rest of the IContainerExtension to achieve it.

Reply:
I'm not concerned with edge cases. Once you open it, you can't close it, and there is no reason to open this yet.